dangerfile.js

```diff
@@ -23,3 +23,8 @@ if (
 ) {
   warn("No CHANGELOG added.");
 }
+
+// If there are changes in the UI and PR is not marked as trivial, expect there to be E2E-test updates
+if (frontendSources.modified && !e2eTestSources.modified && !trivialPR) {
```

Comment:
Asking myself how we should handle changes in the translations or tests. These changes don't require an update in the e2e tests

Reply:
changed, so only changes in the `frontend/src` folder or `frontend/src/pages/` folder trigger the warning. 
also changed the message